addons-l10n/en/project-source-button.json

```diff
@@ -0,0 +1,5 @@
+{
+  "project-source-button/@name": "Project source button",
+  "project-source-button/@description": "Add a button to display JSON files and assets in the project source.",
```

Comment:
I don't think this is necessary since the strings in the `addon.json` are already translated.